src/cpp.d

```diff
@@ -0,0 +1,787 @@
+// Compiler implementation of the D programming language
+// Copyright (c) 1999-2015 by Digital Mars
+// All Rights Reserved
+// written by Walter Bright
```

Comment:
Then just remove the "Written by" ;-)
Btw copyright should go to the "D Language Foundation" ...